Packs/PowershellRemoting/Integrations/PowerShellRemoting/README.md

```diff
@@ -0,0 +1,293 @@
+PowerShell Remoting is a comprehensive built-in remoting subsystem that is a part of Microsoft's native Windows management framework (WMF) and Windows remote management (WinRM).
+This feature allows you to handle most remoting tasks in any configuration you might encounter by creating a remote PowerShell session to Windows hosts and executing commands in the created session.
+The integration includes out-of-the-box commands which supports agentless forensics for remote hosts.
+This integration was integrated and tested with version xx of PowerShell Remoting
```

Comment:
@David-BMS we should  either remove this line or provide something else regarding the setup that was used for testing (for example  Windows machines).

Reply:
removing